.github/workflows/python-package-conda.yml

```diff
@@ -33,6 +33,8 @@ jobs:
             conda install pyparsing
             conda install -c anaconda ipython_genutils 
             conda install -c conda-forge meshplot
+            conda install numba
+            pip install networkx
```

Comment:
I would prefer to keep all dependencies in conda

https://anaconda.org/anaconda/networkx

Reply:
No problem.